scripts/package-build/build.py

```diff
@@ -147,34 +172,38 @@ def cleanup_build_deps(repo_dir: Path) -> None:
 def copy_packages(repo_dir: Path) -> None:
     """Copy generated .deb packages to the parent directory"""
     try:
-        deb_files = glob.glob(str(repo_dir / '*.deb'))
+        deb_files = glob.glob(str(repo_dir / "*.deb"))
         for deb_file in deb_files:
             shutil.copy(deb_file, repo_dir.parent)
             print(f'I: copy generated "{deb_file}" package')
     except Exception as e:
         print(f"Error copying packages: {e}")
 
 
-if __name__ == '__main__':
+if __name__ == "__main__":
     # Prepare argument parser
     arg_parser = ArgumentParser()
-    arg_parser.add_argument('--config',
-                            default='package.toml',
-                            help='Path to the package configuration file')
-    arg_parser.add_argument('--patch-dir',
-                            default='patches',
-                            help='Path to the directory containing patches')
+    arg_parser.add_argument(
+        "--config",
+        default="package.toml",
+        help="Path to the package configuration file",
+    )
+    arg_parser.add_argument(
+        "--patch-dir",
+        default="patches",
+        help="Path to the directory containing patches",
+    )
```

Comment:
Please don't change the format in the same commit as the fix.
Changing the format should be a separate PR or commit. This way, it will be clear what exactly was changed by fix and what was reformatted.

Reply:
PR is closed.